huntsman/tests/test_aardvark.py

```diff
@@ -33,7 +37,49 @@ def test_name_server(name_server):
 def test_locate_name_server(name_server):
     # Check that we can connect to the name server
     Pyro4.locateNS()
-    
+
+
+def test_test_server(test_server):
+    # Check that it is running
+    assert test_server.poll() is None
+
+
+def test_quantity_argument(test_proxy):
+    test_proxy.quantity_argument(550 * u.nm)
+    test_proxy.quantity_argument(1.21 * u.GW)
+
+
+def test_quantity_return(test_proxy):
+    q = test_proxy.quantity_return()
+    assert isinstance(q, u.Quantity)
+    assert q == 42 * u.km * u.ng / u.Ms
+
+
+def test_builtin_exception(test_proxy):
+    with pytest.raises(RuntimeError):
+        test_proxy.raise_runtimeerror()
+    with pytest.raises(AssertionError):
+        test_proxy.raise_assertionerror()
+
+
+def test_pocs_exception(test_proxy):
+    with pytest.raises(error.PanError):
+        test_proxy.raise_panerror()
+
+
+def test_pocs_subclass(test_proxy):
+    with pytest.raises(error.Timeout):
+        test_proxy.raise_timeout()
+    with pytest.raises(error.NotSupported):
+        test_proxy.raise_notsupported()
+    with pytest.raises(error.IllegalValue):
+        test_proxy.raise_illegalvalue()
+
+
+def test_undeserialisable(test_proxy):
```

Comment:
undeserialisable? Is that a word? :grin:

Reply:
It is now!